espnet/asr/pytorch_backend/asr.py

```diff
@@ -490,6 +493,9 @@ def train(args):
 
     # save snapshot which contains model and optimizer states
     trainer.extend(torch_snapshot(), trigger=(1, 'epoch'))
+    if args.save_interval_iters > 0:
```

Comment:
Same comment. I think it is safe to use `if else` instead of set the torch_snapshot twice.